### ISSUE TITLE
Install for Maintenance Solution seems broke on the URL

### ISSUE THREAD

**Issue by wsmelton** [sqlcollaborative/dbatools#3483]
`Install-DbaMaintenanceSolution` seems broke on trying to download from GitHub. Also when the download fails the extraction is trying to still run so fails with red because of the file not existing.

![image](https://user-images.githubusercontent.com/11204251/38375293-84b79140-38bb-11e8-8d12-5afe5cf01041.png)

Command being run:
```powershell
Install-DbaMaintenanceSolution -SqlInstance server -Database master -LogToTable -Solution IndexOptimize
```

Module version: `0.9.321`

**Comment by EnjoyinMoss** [comment]
I've found a workaround for this.  

I made a change to the allcommands.ps1 in module version (0.9.326)

Comment out the Start-BitsTransfer line for the `Install-DbaMaintenanceSolution` and replace with:

```
$wc = New-Object System.Net.WebClient
[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
$wc.DownloadFile($url,$zipfile)
```

Without specifying TLS1.2 the transfer encounters an SCHANNEL error.  Believe the possible cause this has to do with github URL's and Amazon redirects, though could be wrong, didn't explore the full reasons. 

I was unable to get `Start-BitsTransfer` to work even with specifying the security in the manner I did so tried using the .NET WebClient which worked!

NetMon or Ethereal(WireShark) can be helpful if someone wanted to look into this further.
